src/components/Theme/constants.scss

```diff
@@ -43,32 +47,28 @@ $navigation-padding-sides: $padding-lg;
 $navigation-padding-top-sm-screen: $padding-sm;
 $navigation-padding-sides-sm-screen: $padding-xs;
 
+/* COLORS */
+/*
+ *  NOTE: Use rgb() notation instead of rgba() when using Castor basic color tokens.
+ *        The CSS color values do not have an opacity value. Only RGB.
+ *        Color reverts to browser default on IE11 which flags color value as invalid when rgba() used.
+ */
+
 /* Solid colors */
-/*  Use rgb() notation instead of rgba() as these colours do not have an opacity value.
-    Colour reverts to browser default on IE11 which is correctly flagging value as invalid if rgba() used. */
 $color-white: rgb(var(--ods-color-neutral-white));
 $color-black: rgb(var(--ods-color-neutral-black));
 
-$color-icon-temporary: rgba(var(--ods-color-neutral-300));
-$color-divider: rgba(var(--ods-color-neutral-400));
-$color-icons-connector-line: rgba(var(--ods-color-neutral-800));
-$color-body-text: rgba(var(--ods-color-neutral-800));
-$color-title-text: rgba(var(--ods-color-neutral-800));
-$color-subtitle-text: $color-body-text;
+$color-icon-temporary: rgb(var(--ods-color-neutral-300));
+$color-divider: rgb(var(--ods-color-neutral-400));
+$color-icons-connector-line: rgb(var(--ods-color-neutral-800));
 
 /* Transparent colors */
-$color-transparent: transparent;
-$color-help-container: rgba(var(--ods-color-neutral-600));
 $color-modal-overlay: rgba(0, 0, 0, 0.6);
 $color-camera-overlay: rgba(0, 0, 0, 0.7);
 $color-camera-error-overlay: #1c1f21;
 $color-navbar-gradient-one: rgba(28, 31, 33, 0);
 $color-navbar-gradient-two: rgba(28, 31, 33, 0.35);
 
-/* Error message button colors */
-$color-error-button-hover: #ffb9b9;
-$color-error-button-active: #fce5e5;
```

Comment:
Yeap, these constants & the colors are now redundant as they've been replaced in PR #1340 with their CSS variable token equivalent [here](https://github.com/onfido/onfido-sdk-ui/blob/96997d774f67d33725141724d806360c1f2da6ac/src/components/Theme/tokens.scss#L80). 

Castor already has theme & basic color tokens that should match up with the designers' palettes on Figma. At least in the case of the basic color tokens. The theme tokens need the design migration to happen first for us devs to be able to know which to use going forward. I'm planning to create a team wiki page to document this process along with token naming convention rather than overload the UI Customisations project page & probably mention that in the CONTRIBUTING doc.